packages/react-instantsearch-router-nextjs/package.json

```diff
@@ -56,6 +57,7 @@
     "@wdio/cli": "5.16.9",
     "@wdio/jasmine-framework": "5.16.5",
     "@wdio/local-runner": "5.16.9",
+    "@wdio/sauce-service": "5.16.5",
```

Comment:
should we use this for the other e2e tests later too?

Reply:
oh I misread which package.json this was on, I thought it was the root one @dhayab

Reply:
nw, while looking into it I found we have 2 different versions of sauce for no particular reason, so i'll remove the extra version.